app/src/main/java/com/kabouzeid/gramophone/util/MusicUtil.java

```diff
@@ -107,22 +109,45 @@ public static void setRingtone(@NonNull final Context context, final int id) {
     public static String getArtistInfoString(@NonNull final Context context, @NonNull final Artist artist) {
         int albumCount = artist.getAlbumCount();
         int songCount = artist.getSongCount();
-        String albumString = albumCount == 1 ? context.getResources().getString(R.string.album) : context.getResources().getString(R.string.albums);
-        String songString = songCount == 1 ? context.getResources().getString(R.string.song) : context.getResources().getString(R.string.songs);
-        return albumCount + " " + albumString + " • " + songCount + " " + songString;
+
+        return MusicUtil.buildInfoString(
+            MusicUtil.getAlbumCountString(context, albumCount),
+            MusicUtil.getSongCountString(context, songCount)
+        );
+    }
+
+    @NonNull
+    public static String getAlbumInfoString(@NonNull final Context context, @NonNull final Album album) {
```

Comment:
Can you add JavaDocs to these methods with basic examples of what they'd return?

Reply:
Hi, I dont think for such straightforward functions any comment is needed, and if I'm not wrong there is not much javadocs in this project:-)

I'm not against documentation, but it's better when it is done consistently and where the code is not trivial.

Nevertheless, some comment has been added to the new buildInfoString.

Reply:
Hi @arkon, can we move on with this PR?